feat: Support PodDisruptionBudgets (#399)
stackabletech/hbase-operator: docs/modules/hbase/pages/usage-guide/operations/graceful-shutdown.adoc

= Graceful shutdown

Graceful shutdown of HBase nodes is either not supported by the product itself
or we have not implemented it yet.

> The efforts of implementing graceful shutdown for all products that have this functionality is tracked in

Reviewer: ```suggestion
Outstanding implementation work for the graceful shutdowns of all products where this functionality is relevant is tracked in
```
